addons/mrp/models/stock_move.py

```diff
@@ -573,9 +573,9 @@ def _compute_kit_quantities(self, product_id, kit_qty, kit_bom, filters):
                 qty_per_kit = bom_line.product_uom_id._compute_quantity(uom_qty_per_kit / kit_bom.product_qty, bom_line.product_id.uom_id, round=False)
                 if not qty_per_kit:
                     continue
-                incoming_moves = bom_line_moves.filtered(filters['incoming_moves'])
-                outgoing_moves = bom_line_moves.filtered(filters['outgoing_moves'])
-                qty_processed = sum(incoming_moves.mapped('product_qty')) - sum(outgoing_moves.mapped('product_qty'))
+                incoming_qty = sum(bom_line_moves.filtered(filters['incoming_moves']).mapped(lambda m: m.quantity if m.picked else m.product_qty))
+                outgoing_qty = sum(bom_line_moves.filtered(filters['outgoing_moves']).mapped(lambda m: m.quantity if m.picked else m.product_qty))
```

Comment:
in case of `m.picked` you need to convert `m.quantity` in the product's uom. We do not have the computed field for `quantity`

Reply:
I always forget that sorry